### ISSUE TITLE
Project Poster

### ISSUE THREAD

**Issue by mikimn** [Technion236503/2020b-Haifa3D#22]
As part of the course, prerequisites, a poster needs to be made for the app, which will be included in the faculty's software lab (ICST) website, and will bring you eternal credit!

Attached below is a template for the poster. You should edit it, following these criteria:
* ICST, Technion and CS Faculty logos should remain unchanged
* At the bottom, write "Project in Software - Android Application"

* The poster should include a title, overview of the project, features of the application and some screenshots.
* The poster should also include your information (name, email) and your instructor's name
* All content can be written in English

Due date is 28/6, which is strict since posters are sent to the faculty website afterward.

Feel free to ask any poster-related questions here

[DDIOT Poster Pattern 97x67.pptx](https://github.com/Technion236503/2020b-Haifa3D/files/4776416/DDIOT.Poster.Pattern.97x67.pptx)

**Comment by georg-jung** [comment]
[20200625_Haifa3dAppPoster.pdf](https://github.com/Technion236503/2020b-Haifa3D/files/4831782/20200625_Haifa3dAppPoster.pdf)
[20200625_Haifa3dAppPoster.pptx](https://github.com/Technion236503/2020b-Haifa3D/files/4831784/20200625_Haifa3dAppPoster.pptx)

That's my first take on the poster. Probably it doesn't look like a designers masterpiece, but it's a start I guess - feel free to send any feedback.

I chose Jost and Source Sans Pro (available under permissive licenses at Google Fonts) for the text, which I guess are a bit more sleek looking than Arial and Calibri. For simplicity, I attached a PDF too, so that the fonts aren't needed for viewing. If that's against you ideas or you don't like the look, just drop a note.
